netdata.spec.in

```diff
@@ -54,22 +53,34 @@ AutoReqProv: yes
 # Redefine centos_ver to standardize on a single macro
 %{?rhel:%global centos_ver %rhel}
 
-#
 # Conditional build:
+# netns
 %bcond_without  netns    # build with netns support (cgroup-network)
 
 %if 0%{?fedora} || 0%{?rhel} >= 7 || 0%{?suse_version} >= 1140
 %else
 %undefine	with_netns
 %endif
 
+# go.d.plugin
+if [[ -z "${NETDATA_DISABLE_GO+x}" ]]
+  then
+    %bcond_with goplugin
+else
+  %bcond_without goplugin
+fi
+# End conditional build
+
+
```

Comment:
Not sure how I missed this previously.

```suggestion
# go.d.plugin
%bcond_without goplugin
```

RPM will automatically provide the required logic for a CLI option for `rpmbuild` to disable the plugin, and will default to having it enabled.